server/routes/events.js

```diff
@@ -62,4 +62,22 @@ router.delete("/:id", async function (req, res) {
   }
 });
 
+router.post("/:id/attendance", async function (req, res) {
+  try {
+    const { id, userID } = req.params;
+    const event = await Event.findById(id);
+    const user = await User.findById(userID);
+    if (event.attending.contains(user)) {
```

Comment:
I don't see the `userID` parameter being passed in the route definition `'/:id/attendance` - maybe we should add userID as a parameter after attendance? I.e. `'/:id/attendance/:userID`.